spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -46,6 +60,74 @@
   describe "#to_json" do
     subject { described_class.new(generic_file).to_json }
 
-    it { is_expected.to eq json }
+    it do
+      is_expected.to eq json
+    end
+  end
+
+  let(:converter) { InvenioRdmRecordConverter.new }
+  let(:non_user_creator_name) { "I Don't Exist" }
+  let(:personal_creator_without_user_json) {
+    {
+      "creators": [{
```

Comment:
We could probably try to refactor this a bit to save lines, but for a small test it seemed best to keep it simple and clear by outlining exactly what the structure was for each. Also, this is probably going to have to be updated at some point.

Reply:
No big deal for now. I do agree we should start organizing this spec as it grows. It would be nice to refactor this and add more context/describe blocks.